pkg/k8s.io/api/core/v1/types.go

```diff
@@ -1408,7 +1408,7 @@ const (
 	PodPending PodPhase = "Pending"
 	// PodRunning means the pod has been bound to a node and all of the containers have been started.
 	// At least one container is still running or is in the process of being restarted.
-	PodRunning PodPhase = "Running"
+	PodRunning PodPhase = "isRunning"
```

Comment:
The `s/Running/isRunning/` changes look like a too aggressive search-and-replace.